src/sentry/static/sentry/app/routes.jsx

```diff
@@ -139,7 +139,13 @@ function routes() {
         component={errorHandler(LazyLoad)}
       />
 
-      <Route name="Security" path="security/">
+      <Route
+        name="Security"
+        path="security/"
+        componentPromise={() =>
+          import(/*webpackChunkName: "AccountSecurityWrapper"*/ './views/settings/account/accountSecurity/AccountSecurityWrapper')}
```

Comment:
Consider each child route's API endpoints that are required, e.g. `Enroll` needs a different endpoint than `Wrapper`.